doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -516,3 +526,135 @@ arguments used for configuration.  Here is a table describing these emulated dev
+    "<imgfile>:/boot/initrd.img*" in the ``virtio-blk`` input box in the ACRN
+    Configurator. After the ``acrn-dm`` instance exits, ``unmount_partition``
+    will be called to unmount the image file.
+
+``unmount_partition <dir>``
+    Unmount partition from specified directory.
+
+``add_cpus <cpu_apicid>...``
+    Return an ``acrn-dm`` command line parameter fragment to set
+    ``cpu_affinity``. Please refer to `cpu_affinity`_ for details.
+    ``offline_cpus`` is called if the User VM is an RTVM or its scheduler is
+    ``SCHED_NOOP``.
+
+``add_interrupt_storm_monitor <threshold_per_sec> <probe_period_in_sec> <inject_delay_in_ms> <delay_duration_in_ms>``
+    This is added if PCIe devices except integrated GPU are passed through to
+    the User VM to monitor if interrupt storm occurred on those devices.
+    This function and parameters are not visible in the ACRN Configurator and
+    handled by config scripts.
+    It returns acrn-dm command line segment to set ``intr_monitor``.
+
+``add_logger_settings console=<n> kmsg=<n> disk=<n>``
+    set log level of each logging channel. (console, kmsg, disk). This is not
+    exposed to user in the ACRN Configurator.
+
+``add_virtual_device <slot> <kind> <options>``
+    Add specified kind of virtual device to the specified PCIe device slot.
+    Some devices need options to configure further behaviors. <slot> numbers
+    for virtual devices and passthrough devices are automatically allocated
+    by ``launch_cfg_gen.py``.
+
+    Typical use cases:
+
+    - ``hostbridge``
+        PCIe host bridge. <slot> must be 0.
+
+    - ``uart vuart_idx:<int>``
+        Add a PCIe vuart with specified index.
+
+    - ``xhci <bus>-<port>[:<bus>-<port>]...``
+        Config USB mediator. A list of USB ports each specified by
+        ``<bus>-<port>`` will be connected to the User VM.
+
+    - ``virtio-net tap=<tapname>[,vhost],mac_seed=<str>``
+        The TAP should already be created by ``create_tap``.
+
+    - ``virtio-blk <imgfile>[,writethru|writeback|ro]``
+        Add a virtio block device to User VM. The backend is a raw image
+        file. Options can be specified to control access right.
+
+    For all types of virtual devices and options, please refer to
+    :ref:`emul_config`.
+
+``add_passthrough_device <slot> <bus>/<device>/<function> <options>``
+    Passthrough PCIe device to User VM in specified ``<slot>``.
+    Some kinds of devices may need extra ``<options>`` to control internal
+    behavior. Please refer to ``passthru`` section in :ref:`emul_config`.
+
+These functions in template are copied to target launch script. Then
+``launch_cfg_gen.py`` generates following dynamic part. It first defines
+necessary variables like ``vm_type`` and ``scheduler``. Then above functions
+are called to construct the ``dm_params`` per user settings in ``scenario.xml``.
+And last, ``acrn-dm`` is executed to launch a User VM with these parameters.
+
+
+******************
```

Comment:
There's not enough information in this section to include it now, including a usage model for "Dynamic VMs" and a real example with enough details.  I'd recommend this section be deleted and we look into these "Dynamic VMs" after the 3.0 release.  @gvancuts @NanlinXie Your thoughts?

Reply:
Yes, I agree. Further discussions on Dynamic VMs are required.